src/OrbitQt/MoveFilesProcess.cpp

```diff
@@ -24,13 +24,16 @@ void MoveFilesProcess::Start() {
   QMetaObject::invokeMethod(this, [this]() { Run(); });
```

Comment:
Nit: For the current use-case this is not needed, but from a design perspective the user of this class could call `Start` multiple times with the same object. To make this work `interruption_requested_` needs to be reset to false here.

Reply:
Oh good point, done.